src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -213,7 +213,7 @@ public final class Constants {
 
     public static final String HAZELCAST_EXERCISE_CACHE = HAZELCAST_QUIZ_PREFIX + "exercise-cache";
 
-    public static final long MONITORING_SAVE_DELAY = 60 * 30 * 1000; // 30 minutes in milliseconds
+    public static final long MONITORING_CACHE_RESET_DELAY = 60 * 30 * 1000; // 30 minutes in milliseconds
```

Comment:
I'd change the order. Currently the 30 is in the middle between 60 (second -> minute) and 1000 (millisecond -> second)

Reply:
Solved in 591765c28e193caad3be73e1b941d6e2be09a632